test/Maestro/UserRegistration.yml

```
@@ -0,0 +1,58 @@
+    KIDSNAME: Awesome
+
+---
+- launchApp:
+    clearState: true
+
+- tapOn: "E-mail address"
+- inputText: 
+    label: Enter the test user's Email
+    text: ${USERNAME}
+- tapOn:
+    point: "50%,37%"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)